src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -188,6 +189,10 @@ static CPythonEvaluator()
                     {
                         globalScope = CreateGlobalScope();
                     }
+                   
+                    // Reset the 'sys.path' value to the default python paths on node evaluation. 
+                    code = "import sys" + Environment.NewLine + "sys.path = sys.path[0:3]" + Environment.NewLine + code;
```

Comment:
This should be done in its own scope. Otherwise the line numbers will be moved, which will affect traceback.

Reply:
alternatively it can be done from the c# side - without injecting more code into python code string at all.